src/main/resources/application.properties

```
@@ -10,4 +10,6 @@ spring.jpa.hibernate.ddl-auto=validate
 
 #Set Flyway properties
 flyway.url=jdbc:mysql://localhost:3306/mysql
-flyway.schemas =remindme 
+flyway.schemas=remindme
+
+security.jwt.secret-key=remindmesecretkey
```

Use ENV variable and default value 
```suggestion
security.jwt.secret-key=${JWT_SECRET_KEY:<default_key>}
```